### DISCUSSION TITLE
"Tijdstip" doesn't update while it does with "Time" in English version

### DISCUSSION THREAD

**Discussion by marcothijs** [dodona-edu/feedback-and-support#21] [Questions - Vragen] [answered]
"Tijdstip" doesn't update while it does with "Time" in English version.  See picture, it was taken on the 14/10. But the English version gets correctly updated.

![Dodona tijdstip doesn't update](https://github.com/dodona-edu/dodona/assets/60760890/468dc53c-684c-4ed8-a93f-782e233ea6d9)

**Comment by chvp** [comment]
The text is generated when the page is loaded. If you change language on Dodona, the page is reloaded, which means that the text will be updated. If you switch to dutch again, you should see that the text is correct there as well.

**Reply by marcothijs** [reply]
Hi, 

I've checked it again and that is not the case. I still see in the dutch version cases where the "Tijdstip" field doesn't update, where the English version does. Even when I go back to the Dutch version, it still doesn't update, even not when I logged out and in again. Something doesn't look right.

**Reply by bmesuere** [reply]
Can you give us the exact place where this happens, so we can investigate further? (Url + part of the page)

**Comment by marcothijs** [comment, marked as answer]
Hi Bart,

The page was https://dodona.be/nl/courses/2780/series/29335/activities/1724933794/submissions/  but when I checked the page from my Mac it was ok, but from my Windows PC it didn’t refreshed. After a forced refresh (ctrl + reload) it also showed the correct data. It is not something I do normally surfing the net, hence my remark in github.

So it was showing me the cached page instead of the updated page. Any idea why it would go to the browser cache instead of the updated info? Is it a setting on my end or on the server’s end?


Sorry for the inconvenience.
Thanks for your help.

Marco Thijs

From: Bart Mesuere ***@***.***>
Date: Saturday, 21 October 2023 at 10:25
To: dodona-edu/dodona ***@***.***>
Cc: Marco ***@***.***>, Author ***@***.***>
Subject: Re: [dodona-edu/dodona] "Tijdstip" doesn't update while it does with "Time" in English version (Discussion dodona-edu/dodona#5035)

Can you give us the exact place where this happens, so we can investigate further? (Url + part of the page)
